Index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css"/>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+    <title>Imaginary Dance</title>
+  </head>
+  <body>
+    <div class="main">
+      <header>
+        <div class="top-header">
+          <ul class="top-header-list">
+            <li>
+              <a href=""><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a href="">English</a></li>
+            <li><a href="Index.html">My Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </div>
+        <nav>
+          <div class="modal">
+            <img src="./Image/Enabled.svg" class="close" alt="closeTag" />
+            <div class="modal-body">
+              <ul class="modal-menu">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="About.html">About</a></li>
+                <li><a href="#">Feature</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="logo">
+            <img src="./image/logo.png" class="logo-main" alt="logo" />
+          </div>
+          <div class="nav-hamburger">
+            <img src="./image/button.png" class="hamburger" alt="menu-bar" />
+          </div>
+          <div class="menu-icons">
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="#">Feature</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="home" id="home-bg">
+        <div class="home-container">
+          <div class="dance-logo">
+            <img
+              src="https://img.icons8.com/ios-filled/100/000000/ballroom-dance.png"
+              alt="dancing-logo"
+            />
+          </div>
+          <div class="title">
+            <h1>"The Imaginary Dance"</h1>
+            <h2>Bringing life through dance</h2>
+          </div>
+          <div class="home-info">
+            <p>
+              “Stars are always dancing. Sometimes they dance twinkling away
+              with the rhythm of your joyful heart and sometimes they dance
+              without movement to embrace your heartache as if frozen sculptures
+              of open-armed sadness.”
+            </p>
+          </div>
+          <div class="home-info-2">
+            <p>
+              This is why the idea of finding love across the dance floor
+              endures – symbolizing that, when we know the true rhythm of our
+              heart, we know the other.
+            </p>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] The `hero` section (first section of the home page) should fill the whole screen view on `desktop` sized screens, the `font-size` is a bit smaller compared to the design. Kindly update the `hero` section to match the design.
Reference Design
![image](https://user-images.githubusercontent.com/33767951/142095400-48f610cc-bfe5-401d-bf61-3abf3148a038.png)
Your Styling
![image](https://user-images.githubusercontent.com/33767951/142337515-c80c8d57-21ad-4b43-98ff-e9620f3d9a53.png)